components/support/ktx/src/main/java/mozilla/components/support/ktx/android/content/Context.kt

```diff
@@ -79,3 +80,25 @@ fun Context.share(text: String, subject: String = getString(R.string.mozac_suppo
         false
     }
 }
+
+/**
+ *  Returns true if we are running in the main process false otherwise.
+ */
+fun Context.isMainProcess(): Boolean {
```

Comment:
@pocmo 
My initial idea was to cache the value as we chatted, but there some [limitations with lazy ](https://youtrack.jetbrains.com/issue/KT-9686) [[2]](https://discuss.kotlinlang.org/t/lazy-extension-properties/2351) that make it tricky to do: 
```
val Context.isMainProcess : Boolean by lazy {
    // I can't access Context@this here.
}
```